yktracec/src/jitmodbuilder.cc

```diff
@@ -940,11 +941,13 @@ class JITModBuilder {
 
     // Get the instruction index of CPCI in its parent block.
     size_t CPCIIdx = 0;
+    assert(CPCI->getParent() != NULL);
     for (Instruction &I : *CPCI->getParent()) {
       if (&I == cast<Instruction>(CPCI)) {
         break;
       }
       CPCIIdx++;
+      assert(CPCI->getParent() != NULL);
```

Comment:
Line 945 would have crashed before this assertion could be checked. Move it up?

It's actually impossible that `CPCI` is unparented, since it's been extracted from the AOT module. No instruction in the AOT module is unparented by construction.

I'd just remove this assertion personally.